test/integration/request_test.rb

```diff
@@ -37,6 +37,9 @@ def test_request_times
     assert_equal 1, aggregate["rack.request.time"][:count],
       'should track total request time'
 
+    # should calculte p95 value
+    refute_equal aggregate.fetch("rack.request.time", percentile: 95), 0.0
```

Comment:
Hmm, this reads a bit mind-bendy to me, maybe because I rarely use `refute_equal`. Maybe:

```rb
assert aggregate.fetch("rack.request.time", percentile: 95) > 0.0
```
?

Also small type s/calculte/calculate